arcgis_map_sdk_method_channel/lib/src/method_channel_arcgis_map_plugin.dart

```diff
@@ -34,6 +34,36 @@ class MethodChannelArcgisMapPlugin extends ArcgisMapPlatform {
     throw UnimplementedError('setMouseCursor() has not been implemented');
   }
 
+  @override
+  void setAutoPanMode(String autoPanMode, int mapId) {
+    _methodChannelBuilder(mapId).invokeMethod("set_auto_pan_mode", autoPanMode);
+  }
+
+  @override
+  Future<AutoPanMode> getAutoPanMode(int mapId) {
+    return _methodChannelBuilder(mapId)
+        .invokeMethod<String>("get_auto_pan_mode")
+        .then(
+          (value) =>
+              AutoPanMode.values.firstWhere((element) => element.name == value),
+        );
```

Comment:
Does this work?
```suggestion
        .then(AutoPanMode.values.byName);
```

Reply:
`(value)` is nullable. I need to fallback in this case 🤔